src/StreamJsonRpc/MessageFormatterHelper.cs

```diff
@@ -0,0 +1,31 @@
+﻿
+namespace StreamJsonRpc
+{
+    using System;
+    using System.Linq;
+    using System.Reflection;
+
+    /// <summary>
+    /// Class containing useful method to help on the implementation of message formatters.
+    /// </summary>
+    public class MessageFormatterHelper
+    {
+        /// <summary>
+        /// Converts given <see cref="Type"/> to its IProgress <see cref="Type"/>.
```

Comment:
>IProgress <see cref="Type"/> [](start = 53, length = 28)

The link in the doc would be better to go to the type we're talking about, rather than System.Type:
`<see cref="IProgress{T}" />`